templates/components/search/display_data.html.twig

```diff
@@ -52,15 +52,6 @@
             {% endif %}
          {% else %}
             {{ include('components/search/controls.html.twig') }}
-
```

Comment:
The new limit dropdown triggers a full page reload instead of using the fluid search feature because of missing properties.
Patch:
```diff
diff --git a/templates/components/pager.html.twig b/templates/components/pager.html.twig
index 63795132c9..d78cb1396a 100644
--- a/templates/components/pager.html.twig
+++ b/templates/components/pager.html.twig
@@ -66,7 +66,10 @@
 {% set skip_adjacents = false %}
 
 <div class="flex-grow-1 d-flex flex-wrap flex-md-nowrap  align-items-center justify-content-between mb-2 search-pager">
-   {% set limitdropdown = include('components/dropdown/limit.html.twig') %}
+   {% set limitdropdown = include('components/dropdown/limit.html.twig', {
+      'on_change': true,
+      'select_class': 'search-limit-dropdown',
+   }) %}
    <span class="search-limit d-none d-md-block">
       {{ __('%s rows / page')|format(limitdropdown)|raw }}
    </span>
```